src/plugins/vis_type_timelion/server/routes/run.ts

```diff
@@ -79,6 +79,16 @@ export function runRoute(
       try {
         const uiSettings = await context.core.uiSettings.client.getAll();
 
+        const getIndexPatternsService = async () => {
+          // @ts-ignore need help resolving this one
```

Comment:
to fix that, please update type in line 36  `CoreSetup -> CoreSetup<TimelionPluginStartDeps>` 

![image](https://user-images.githubusercontent.com/20072247/106259465-55a8a500-6230-11eb-90e7-ca40d343d808.png)